### ISSUE TITLE
History primary key issue

### ISSUE THREAD

**Issue by pzac** [ifad/chronomodel#79]
`@project.history.first.update_column(:foo, 'bar')` will update all history records with the same `id`, instead of using `hid` as primary key.

**Comment by vjt** [comment]
This causes data loss on the history and will be fixed ASAP.

**Comment by pzac** [comment]
Additional info: 
```
development: main:0> @project.history.last.id
=> 66847
development: main:0> @project.history.last[:id]
=> 74
```
Redefining the `id` can have unwanted side-effects: who knows which methods within ActiveRecord use `x.id` or `x[:id]`

**Comment by pzac** [comment]
https://github.com/ifad/chronomodel/blob/master/lib/chrono_model/time_machine.rb#L71

`update_columns` too, and probably more methods

https://github.com/rails/rails/blob/1198a3880bb2918bb771b55d94cfb333813744fa/activerecord/lib/active_record/persistence.rb#L484

**Comment by vjt** [comment]
Fixed by #82, thanks to @pzac.
